features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
```

Comment:
The use of "objects" is a bit tricky here: Are you talking about physical objects or object-oriented-programming objects?